packages/engine/errors/1208.md

```diff
@@ -0,0 +1,7 @@
+---
+original: "'{0}' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module."
+excerpt: "You have set the 'isolatedModules' flag. In the 'A' file you write certain code that can’t be correctly interpreted by a single-file transpilation process. Add an import, export, or an empty 'export {}' statement to make it a module."
```

Comment:
> In the 'A' file you write certain code that can’t be correctly interpreted by a single-file transpilation process

Could you explain this in a clearer way? Perhaps using less complicated language (interpreted, transiplation)

Reply:
Okay, I'll think about how to explain this